src-db/database/model/functions/C_ORDER_POST1.xml

```diff
@@ -1435,6 +1435,32 @@
                   FROM m_reservation
                   WHERE c_orderline_id = cur_orderline.c_orderline_id
                 );
+				
+                --C_Order_Post - Stock Reservation Extension Point
+                SELECT count(*) INTO v_count
+                FROM DUAL
+                where exists (select 1 from ad_ep_procedures where ad_extension_points_id = 'A179E12755784A8885653FE33468B11F');
+                IF (v_count=1) THEN
+                  DECLARE
+                    v_ep_instance VARCHAR(32);
+                    v_extension_point_id VARCHAR(32) := 'A179E12755784A8885653FE33468B11F';
```

Comment:
Maybe this var can be declared before and also you can use it to calculate v_count

Reply:
I did it this way because I followed the convention in how extension points were handled within the “c_order_post” table, likewise, would you say to change only mine even though the others are handled like this?

Reply:
It's good of this way, that is only a suggest.